lib/progress_bar/styles.rb

```diff
@@ -0,0 +1,74 @@
+class ProgressBar
```

Comment:
So Im aware and have used gems which do this for you. But I didnt want to introduce a new dependency and this allows for more internal control by the library as to working styles.

I tested this on Ubuntu 18.04 and Mac OSX (Iterm2).